### DISCUSSION TITLE
mdSilo Desktop v0.5.8

### DISCUSSION THREAD

**Discussion by danloh** [mdSilo/mdSilo-app#655] [Announcements]
bug fixes and samll improvements.

<hr /><em>This discussion was created from the release <a href='https://github.com/mdSilo/mdSilo-app/releases/tag/app-v0.5.8'>mdSilo Desktop v0.5.8</a>.</em>

**Comment by superyngo** [comment]
Nice work especially with the view features.

Suggestions:
1. Remove ![the up-left corner light bulb ](https://github.com/user-attachments/assets/a8bf0062-5bb4-4e99-b9c9-7982fc05f1e6)
icon and move menu bar up.
2. Move ![up bar ](https://github.com/user-attachments/assets/0dfbf55a-3f81-461a-b733-377ed5910cb7) and its function to some where else.

Basically I suggest to let ![this block](https://github.com/user-attachments/assets/bbcefd2f-a5f5-428c-b8f4-be5f7765f153)
vanish for better looking and less visual distraction for functionality hinting.
